### DISCUSSION TITLE
TypeError: Cannot read properties of null (reading 'useEffect')

### DISCUSSION THREAD

**Discussion by saberap** [storybookjs/storybook#20316] [Help]
Hi every one.
I have a problem with using react hooks in storybook.
When I write component and use react hooks every thing are ok but when I build package with Rollup and push it to npm and use this package in my projects I see this error in console:
_TypeError: Cannot read properties of null (reading 'useEffect')_

**Comment by ZhengRui** [comment]
having the same issue after upgrading to storybook v7
